options.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+var permissions = { origins: ["<all_urls>"] };
+var prefPrependSnowflake = document.getElementById("prefPrependSnowflake");
+prefPrependSnowflake.onchange = async () => {
+  if (prefPrependSnowflake.checked) {
+    prefPrependSnowflake.disabled = true;
+    // Note: permissions.request only works when this page is shown in a tab,
+    // i.e. when options_ui.open_in_tab=true in manifest.json.
+    // See https://bugzilla.mozilla.org/show_bug.cgi?id=1382953
```

Comment:
Once this has landed, I'd like us to track this in an issue for the repo here.

Reply:
This upstream bug is going to land soon, possibly fixed in Firefox 61.